<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>751</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QTextEdit" name="textEdit_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>311</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>140</y>
       <width>331</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxIntercept">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>92</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBrowser">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>88</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Browser</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSend">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>60</y>
       <width>88</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
